sycl/include/sycl/ext/oneapi/experimental/bfloat16_math.hpp

```diff
@@ -62,13 +62,26 @@ template <size_t N> sycl::marray<bool, N> isnan(sycl::marray<bfloat16, N> x) {
 template <typename T, int N = num_elements_v<T>>
 std::enable_if_t<is_vec_or_swizzle_bf16_v<T>, sycl::vec<int16_t, N>>
 isnan(T x) {
+
+#if defined(__SYCL_DEVICE_ONLY__) && (defined(__SPIR__) || defined(__SPIRV__))
+  // Convert BFloat16 vector to float vec and call isnan()
+  sycl::vec<float, N> FVec =
+      x.template convert<float, sycl::rounding_mode::automatic>();
+  auto res = isnan(FVec);
```

Comment:
This should be `Res` but I see the rest of the file using `res`, so honestly I'm not sure what's the best way to proceed here. @intel/llvm-reviewers-runtime any opinion?

Reply:
Applies to other occurrences as well.

Reply:
> This should be `Res` but I see the rest of the file using `res`, so honestly I'm not sure what's the best way to proceed here. @intel/llvm-reviewers-runtime any opinion?

There's an inconsistency in the variable naming convention used in this file. Since LLVM suggests using CamelCase for variable names (https://www.llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly), I could use `Res` instead.

Reply:
I'd like it better if you did, at least let's not introduce any more errors. Thanks!